.github/workflows/release.yml

```diff
@@ -6,6 +6,10 @@ on:
       tag:
```

Comment:
We should rename this variable to `ref` maybe, that way we can pass tags or branches. It's a bit strange to be checking if the `tag == main`

Reply:
Makes sense. Updated the code accordingly.